src/commands/sync.ts

```diff
@@ -66,6 +60,10 @@ export class SnowballSync extends Command {
 
   static args = [{ name: 'manifest', required: true }];
   async run(): Promise<void> {
+    // Set back off properties for the uploads
+    BackOff.time = 500;
```

Comment:
I dont think we should override these here, leave them in the `BackOff` Config file